src/redismodule.h

```diff
@@ -1205,6 +1207,7 @@ REDISMODULE_API int (*RedisModule_RegisterEnumConfig)(RedisModuleCtx *ctx, const
 REDISMODULE_API int (*RedisModule_LoadConfigs)(RedisModuleCtx *ctx) REDISMODULE_ATTR;
 
 #define RedisModule_IsAOFClient(id) ((id) == UINT64_MAX)
+#define RedisModule_Assert(_e) ((_e)?(void)0 : (RedisModule__Assert(#_e,__FILE__,__LINE__),exit(1)))
```

Comment:
this change because:
```
[root@binblog redis]# ./runtest-moduleapi
make: Entering directory '/root/redis/self/redis/tests/modules'
gcc -I../../src  -W -Wall -Wno-missing-field-initializers -fno-common -g -ggdb -std=c99 -O2 -fPIC -c commandfilter.c -o commandfilter.xo
In file included from commandfilter.c:1:0:
../../src/redismodule.h: In function ‘RedisModule_Init’:
../../src/redismodule.h:1545:5: warning: implicit declaration of function ‘RedisModule_Assert’; did you mean ‘RedisModule__Assert’? [-Wimplicit-function-declaration]
     RedisModule_Assert(REDISMODULE_GET_API_RET);
     ^~~~~~~~~~~~~~~~~~
     RedisModule__Assert
```